docs/asdf/extending/extensions.rst

```diff
@@ -154,6 +154,26 @@ Tag handles can be defined in the ``yaml_tag_handles`` property of an extension:
         extension_uri = "asdf://example.com/example-project/extensions/foo-1.0.0"
         yaml_tag_handles = {"!example!": "asdf://example.com/example-project/tags/"}
 
+Additional schema validators
+----------------------------
+
+Schema validators implement the `Validator` interface
+and are included in an extension via the ``validators`` property:
+
+.. code-block:: python
+
+    from asdf.extension import Extension, Validator
+
+
+    class FooValidator(Validator):
+        # ...
+        pass
+
+
+    class FooExtension(Extension):
+        extension_uri = "asdf://example.com/example-project/extensions/foo-1.0.0"
+        validators = [FooValidator()]
+
```

Comment:
Should we link to the extending validators or other documentation here in case the reader wants to know more?